docs/tutorial/navigation-history.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Navigation History"
+description: "The NavigationHistory API allows you to manage and interact with the browsing history of your Electron application."
+slug: navigation-history
+hide_title: false
+---
+
+# Navigation History
```

Comment:
I think if users are reading this example when its released, they are probably using the new API? Also the deprecation warnings I added should prompt them to switch..

I was going to add something like below but realized that it's kinda redundant with the part right below on accessing NavigationHistory:
**Note:** Since [Electron v32](https://www.electronjs.org/docs/latest/breaking-changes#deprecated-clearhistory-cangoback-goback-cangoforward-goforward-gotoindex-cangotooffset-gotooffset-on-webcontents), navigation-related APIs have been moved to the `navigationHistory` property of `WebContents` to provide a more structured and intuitive interface for managing navigation history.